ext/random/tests/03_randomizer/user_unsafe.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+Random: Randomizer: User: Engine unsafe
+--FILE--
+<?php
+
+$randomizer = (new \Random\Randomizer(
+    new class () implements \Random\Engine {
+        public function generate(): string
+        {
+            return '';
+        }
+    }
+));
+
+try {
+    $randomizer->getInt(\PHP_INT_MIN, \PHP_INT_MAX);
+} catch (\RuntimeException $e) {
+    echo "catched\n";
```

Comment:
Verify the exception message to make sure you're catching the correct exception. The same applies below.

Reply:
fixed ce5920ec5d
Sorry for the delay.